packages/bits/demo/src/components/demo/textbox/textbox-visual-test/textbox-visual-test.component.html

```diff
@@ -99,5 +99,17 @@ <h3>Textbox with Textarea and Placeholder</h3>
             </nui-textbox>
             <hr />
         </div>
+        <div class="w-100"></div>
+        <div class="col">
+            <h3>Textbox with scrollable Textarea</h3>
```

Comment:
It might be good to also have an automation test for non-scrollable to ensure that consumers can still turn it off if necessary.